content/events/2020-amsterdam/_index.md

```diff
@@ -0,0 +1,86 @@
+    logo: sponsors/container-solutions.svg
+    url: https://www.container-solutions.com/
+  - name: Fullstaq
+    logo: sponsors/fullstaq.png
+    url: https://fullstaq.com/
+speakers:
+# - name: Phippy
+#   image: team/phippy-color.png
+#   twitter: https://twitter.com/cloudnativefdn
+#   bio: Phippy is a simple PHP app, trying to find a home in a cloud native world.
+#   homepage: https://phippy.io
+organizers:
+- name: Alessandro Vozza
+  image: team/alessandro_vozza.jpeg
+  twitter: https://twitter.com/bongo
+  homepage: https://www.linkedin.com/in/alessandrovozza
+- name: Fabio Tiriticco
+  image: team/fabio_tiriticco.jpeg
+  twitter: https://twitter.com/ticofab
+  homepage: https://twitter.com/ticofab
+- name: Daniel Paulus
+  image: team/daniel_paulus.jpeg
+  twitter: https://twitter.com/dpnl87
+  homepage: https://twitter.com/dpnl87
+- name: Floor Drees
+  image: team/floor_drees.jpeg
+  twitter: https://github.com/floord
+  homepage: https://github.com/floord
+- name: Walter Heck
+  image: team/walter_heck.jpeg
+  twitter: https://github.com/walterheck
+  homepage: https://github.com/walterheck
+- name: Serkan Capkan
+  image: team/serkan_capkan.jpeg
+  twitter: https://github.com/srknc
+  homepage: https://github.com/srknc
+- name: Samuel Attal 
+  image: team/samuel_attal.jpeg
+  twitter: https://twitter.com/AttalSam
+  homepage: https://twitter.com/AttalSam
+organizers_email: organizers-amsterdam@kubernetescommunitydays.org
+local_law_enforcement: 112
+medical_emergency: 112
+cfp_deadline: '2020-07-01'
+sponsor_deadline: '2020-09-14'
+---
+
+Kubernetes Community Day comes back to Amsterdam after a great first edition and on the trail of [KubeCon EU Amsterdam](https://events19.linuxfoundation.org/events/kubecon-cloudnativecon-europe-2020/). Check the [videos](https://www.youtube.com/playlist?list=PLQGLXxvf53b0fzCwFJSLRyn88bCoFNH2G) and the [photos](https://500px.com/cloudnativeams/galleries/kubernetes-community-day-amsterdam-2019) of last year, it was a lot of fun!
+
+For 2020, we are going slightly bigger in attendance while mantaining the grassroot spirit and the community feeling that is our underlying theme; we selected a bigger, more spacious venue and we extended the program to 2 full days of technical talks, networking and bonding.
+
+You will be able to buy a ticket for one or two days [here](hhttps://kubernetescommunitydays.org/) soon! You'll be able to buy a discounted early bird ticket before we announce the full lineup, then it's regular price (it's still quite cheap!).
+
+We will soon announce our [speaker](speakers)'s lineup; the [call for papers](cfp) will also go live mid-February.
+Our event would not be possible without support of our [sponsors](sponsor); prospective sponsors can review our [prospectus](prospectus) and get in touch with us via email. We're still finalizing our [program](program) but we're excited to have finalized a great [venue](venue). Please [contact](contact) the organizers with any questions.
```

Comment:
Please specify whether talks can be in either Dutch or English.

Reply:
addressed in 7e2cd5e